api/security/csrf/csrf.py

```diff
@@ -35,7 +35,7 @@ def is_csrf_enabled():
     return os.getenv("ENABLE_CSRF") != 'false'
 
 def set_csrf_cookie(resp, csrf_token):
-    resp.set_cookie(CSRF_COOKIE_NAME, value=csrf_token)
+    resp.set_cookie(CSRF_COOKIE_NAME, value=csrf_token, httponly=True, secure=True, samesite="Lax")
```

Comment:
improvement: it would be important to test the cookie attributes as well (I think we test them in other cookies) since those attributes are important for security

Reply:
Done in next revision! Thanks